packages/new-block-stream/README.md

```diff
@@ -4,6 +4,87 @@ A tool for creating a stream of new block data.
 
 ## Usage
 
+### Getting started
+
+Install via npm:
+
+```js
+npm install @drizzle-utils/new-block-stream
+```
+
+Usage example (taken from `@drizzle-utils/new-block-stream/example.js`)
+
 ```js
-// TODO
+/* eslint-disable no-console */
+const Web3 = require("web3");
+const createNewBlock$ = require("@drizzle-utils/new-block-stream");
+
+const main = async () => {
+  // const web3 = new Web3("http://127.0.0.1:9545"); // HttpProvider
+  const web3 = new Web3("ws://127.0.0.1:9545"); // WebsocketProvider
+
+  const { observable, subscription } = createNewBlock$({
+    web3,
+    pollingInterval: 200, // only used if non-WebsocketProvider
+  });
+
+  // log out new blocks (might skip blocks if polling)
+  const stream = observable.subscribe(console.log);
```

Comment:
I think we can end the usage example here with:

```js
observable.subscribe(block => console.log(block));
```

The reason for the non-point-free form is to make it clearer to the reader that what's getting passed into the function is a block object.